frontend/src/pages/LineageView.tsx

```diff
@@ -74,29 +88,39 @@ class LineageView extends Page<{}, LineageViewState> {
         {title: 'Some Process', desc: '13,201 Examples', prev: true, next: true}
       ]},
     ] as CardDetails[];
+    const targetTitle = this.state.target ? getResourceProperty(this.state.target, ArtifactProperties.NAME) : '';
+    const mockTarget = [...mockExec];
+    mockTarget[0].elements[0].title = targetTitle ? targetTitle as string : '';
     return (
-      <div className={classes(commonCss.page, 'LineageExplorer')} style={{flexFlow: 'row', overflow: 'auto', width: '100%', position: 'relative', background: '#f3f2f4', zIndex: 0}}>
-        <LineageCardColumn
-          type='artifact'
-          cards={mockInputArtifacts}
-          title={`${columnNames[0]}`} />
-        <LineageCardColumn
-          type='execution'
-          cards={mockExec}
-          title={`${columnNames[1]}`} />
-        <LineageCardColumn
-          type='artifact'
-          cards={[Object.assign({}, mockExec[0], {title: 'Target'})]}
-          title={`${columnNames[2]}`} />
-        <LineageCardColumn
-          type='execution'
-          cards={mockExec}
-          title={`${columnNames[3]}`} />
-        <LineageCardColumn
-          type='artifact'
-          cards={mockOutputArtifacts}
-          reverseBindings={true}
-          title={`${columnNames[4]}`} />
+      <div className={classes(commonCss.page)}>
```

Comment:
I see that the CSS is showing in the cluster page. But where was it moved to?

Reply:
Nested below the action bar with the card column container

Reply:
Gotcha, thanks!